StreamChatClient_v3/WebSocketClient/Engine/StarscreamWebSocketEngine.swift

```diff
@@ -22,7 +22,9 @@ final class StarscreamWebSocketProvider: WebSocketEngine {
   }
 
   func connect() {
-    webSocket.connect()
+    DispatchQueue.main.async {
+      self.webSocket.connect()
+    }
```

Comment:
`DispatchQueue.main.async(webSocket.connect)` doesn't work?

Reply:
+1 should work

Reply:
I'm not sure we have to call it as main.async.
@VojtaStavik did you try just `webSocket.connect()`?

Reply:
We talked about this here https://getstream.slack.com/archives/G012PJZJ39C/p1591274843003100?thread_ts=1591274843.003100. I also think it's not necessary. I did it based on your advice 🤷‍♂️

Reply:
@VojtaStavik yes and you said you will call it directly 😆

Reply:
I guess I chickened out. I removed it.